### ISSUE TITLE
[Bug]: Hitting Cmd+X when window is not fully loaded causes some sort of phantom window

### ISSUE THREAD

**Issue by imageslr** [hovancik/stretchly#999]
### Version

- [X] I'm using version 1.7.0

### Known issues

- [X] I've checked Known issues

### Existing issues

- [X] I've checked Existing issues

### What operating system are you using?

macOS

### Operating System Version

10.15.7 (19H2)

### Reproduction steps

1. Sketchly settings -> Full screen
2. Lock screen **10 seconds** before Sketchly's break 
3. Unlock screen after 1 minutes
...
**Reproduces how often:** [What percentage of the time does it reproduce?]
100%

### Expected Behavior

No sketchly's fullscreen break view. I can click something, and switch desktop.

### Actual Behavior

No sketchly's fullscreen break view. I can see my desktop. But I can't click anything, neither can I switch desktop. It's like there is an invisible mask on my screen which prevented me from clicking desktop.

When I exited Sketchly, I can click and everything works fine.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hovancik** [comment]
Hi, could you please paste the log from that time?

**Comment by imageslr** [comment]
Hi，the time is about 17:25~17:29.

```
[2021-07-13 17:15:54.838] [info]  Stretchly: showing Mini Break notification
[2021-07-13 17:16:05.315] [info]  Stretchly: showing window 2 of 2
[2021-07-13 17:16:05.374] [info]  Stretchly: showing window 1 of 2
[2021-07-13 17:16:05.384] [info]  Stretchly: starting Mini Break
[2021-07-13 17:16:08.152] [info]  Stretchly: postponing Mini Break
[2021-07-13 17:17:58.157] [info]  Stretchly: showing Mini Break notification
[2021-07-13 17:18:08.555] [info]  Stretchly: showing window 1 of 2
[2021-07-13 17:18:08.559] [info]  Stretchly: starting Mini Break
[2021-07-13 17:18:08.620] [info]  Stretchly: showing window 2 of 2
[2021-07-13 17:18:08.785] [info]  System: suspend or lock
[2021-07-13 17:18:08.814] [info]  Stretchly: finishing Mini Break
[2021-07-13 17:18:08.820] [info]  Stretchly: pausing breaks for 1
[2021-07-13 17:19:19.028] [info]  System: suspend or lock
[2021-07-13 17:21:57.591] [info]  System: resume or unlock
[2021-07-13 17:21:57.593] [info]  Stretchly: resuming breaks
[2021-07-13 17:22:06.405] [info]  System: resume or unlock
[2021-07-13 17:22:31.244] [info]  Stretchly: reseting breaks
[2021-07-13 17:22:37.102] [info]  System: suspend or lock
[2021-07-13 17:22:37.103] [info]  Stretchly: pausing breaks for 1
[2021-07-13 17:22:38.688] [info]  System: resume or unlock
[2021-07-13 17:22:38.694] [info]  Stretchly: resuming breaks
[2021-07-13 17:23:08.791] [info]  Stretchly: initializing...
[2021-07-13 17:23:08.800] [info]  Stretchly: loading preferences
[2021-07-13 17:23:08.810] [info]  Stretchly: starting Idle time monitoring
[2021-07-13 17:23:08.811] [info]  Stretchly: starting Do Not Disturb monitoring
[2021-07-13 17:26:01.170] [info]  Stretchly: setting 'language' to 'en' (was 'zh-CN')
[2021-07-13 17:26:06.025] [info]  Stretchly: setting 'language' to 'zh-CN' (was 'en')
[2021-07-13 18:02:58.791] [info]  Stretchly: showing Mini Break notification
[2021-07-13 18:03:09.351] [info]  Stretchly: showing window 1 of 2
[2021-07-13 18:03:09.355] [info]  Stretchly: starting Mini Break
[2021-07-13 18:03:09.547] [info]  Stretchly: showing window 2 of 2
[2021-07-13 18:03:11.587] [info]  Stretchly: postponing Mini Break
[2021-07-13 18:05:01.594] [info]  Stretchly: showing Mini Break notification
[2021-07-13 18:05:12.118] [info]  Stretchly: showing window 1 of 2
[2021-07-13 18:05:12.124] [info]  Stretchly: starting Mini Break
[2021-07-13 18:05:12.226] [info]  Stretchly: showing window 2 of 2
[2021-07-13 18:05:13.757] [info]  Stretchly: finishing Mini Break
[2021-07-13 18:05:48.181] [info]  System: suspend or lock
[2021-07-13 18:05:48.183] [info]  Stretchly: pausing breaks for 1
[2021-07-13 18:06:56.425] [info]  System: suspend or lock
[2021-07-13 18:08:03.745] [info]  System: resume or unlock
[2021-07-13 18:08:04.458] [info]  Stretchly: resuming breaks
[2021-07-13 18:08:05.550] [info]  System: resume or unlock
[2021-07-13 18:47:54.512] [info]  Stretchly: showing Mini Break notification
[2021-07-13 18:48:04.992] [info]  Stretchly: showing window 1 of 2
[2021-07-13 18:48:04.995] [info]  Stretchly: starting Mini Break
[2021-07-13 18:48:05.144] [info]  Stretchly: showing window 2 of 2
[2021-07-13 18:48:08.440] [info]  System: suspend or lock
[2021-07-13 18:48:08.455] [info]  Stretchly: finishing Mini Break
[2021-07-13 18:48:08.466] [info]  Stretchly: pausing breaks for 1
[2021-07-13 18:49:09.335] [info]  System: suspend or lock
```

**Comment by hovancik** [comment]
Hi, I don't see anything like that around that time. Can try watching for this when it hapens again and paste the log again?  

The closest I see is:

```
[2021-07-13 17:17:58.157] [info]  Stretchly: showing Mini Break notification
[2021-07-13 17:18:08.555] [info]  Stretchly: showing window 1 of 2
[2021-07-13 17:18:08.559] [info]  Stretchly: starting Mini Break
[2021-07-13 17:18:08.620] [info]  Stretchly: showing window 2 of 2
[2021-07-13 17:18:08.785] [info]  System: suspend or lock
[2021-07-13 17:18:08.814] [info]  Stretchly: finishing Mini Break
[2021-07-13 17:18:08.820] [info]  Stretchly: pausing breaks for 1
[2021-07-13 17:19:19.028] [info]  System: suspend or lock
[2021-07-13 17:21:57.591] [info]  System: resume or unlock
[2021-07-13 17:21:57.593] [info]  Stretchly: resuming breaks
[2021-07-13 17:22:06.405] [info]  System: resume or unlock
```

where you locked the screen around the same time as the break started. That caused Stretchly to finish the break as it pauses breaks when system goes to suspend/lock. 

Do you think that is the case? I can't really reproduce this specific one as there is less then a second time difference.

**Comment by imageslr** [comment]
Thanks for your response. If there were error again I would paste logs asap.

**Comment by imageslr** [comment]
Hi @hovancik I just encountered this issue again.

Here is my log with comments:

```
[2021-07-21 11:20:28.720] [info]  Stretchly: showing Mini Break notification
[2021-07-21 11:20:39.259] [info]  Stretchly: showing window 3 of 3
[2021-07-21 11:20:39.269] [info]  Stretchly: showing window 1 of 3
[2021-07-21 11:20:39.279] [info]  Stretchly: starting Mini Break
[2021-07-21 11:20:39.328] [info]  Stretchly: showing window 2 of 3
[2021-07-21 11:20:40.712] [info]  Stretchly: postponing Mini Break
[2021-07-21 11:22:30.714] [info]  Stretchly: showing Mini Break notification

# issue begins
[2021-07-21 11:22:41.135] [info]  Stretchly: showing window 1 of 3 
[2021-07-21 11:22:41.139] [info]  Stretchly: starting Mini Break
[2021-07-21 11:22:41.268] [info]  Stretchly: showing window 2 of 3
[2021-07-21 11:22:41.293] [info]  Stretchly: showing window 3 of 3

# once the break is showing (fullscreen), I pressed `cmd+x` immediately. They disappeared.
[2021-07-21 11:22:42.152] [info]  Stretchly: finishing Mini Break

# But I found I can't click anything on the window. 
# So I tried to restart a break and hope to remove any element that Stretchly added on my screen.
[2021-07-21 11:22:55.471] [info]  Stretchly: reseting breaks
[2021-07-21 11:22:57.565] [info]  Stretchly: skipping to Long Break
[2021-07-21 11:22:58.084] [info]  Stretchly: showing window 1 of 3
[2021-07-21 11:22:58.088] [info]  Stretchly: starting Mini Break
[2021-07-21 11:22:58.094] [info]  Stretchly: showing window 2 of 3
[2021-07-21 11:22:58.327] [info]  Stretchly: showing window 3 of 3
[2021-07-21 11:23:00.066] [info]  Stretchly: postponing Long Break

# But it didn't work. So I quitted Stretchly. 

# Then I restarted it.
[2021-07-21 11:23:29.056] [info]  Stretchly: initializing...
[2021-07-21 11:23:29.067] [info]  Stretchly: loading preferences
[2021-07-21 11:23:29.154] [info]  Stretchly: starting Idle time monitoring
[2021-07-21 11:23:29.157] [info]  Stretchly: starting Do Not Disturb monitoring
```

**Comment by imageslr** [comment]
If I press `cmd + x` to skip the break very quickly, this will happen again:

```
[2021-07-21 15:30:11.499] [info]  Stretchly: showing Mini Break notification
[2021-07-21 15:30:22.042] [info]  Stretchly: showing window 2 of 3
[2021-07-21 15:30:22.095] [info]  Stretchly: showing window 3 of 3
[2021-07-21 15:30:22.181] [info]  Stretchly: showing window 1 of 3
[2021-07-21 15:30:22.194] [info]  Stretchly: starting Mini Break
[2021-07-21 15:30:23.245] [info]  Stretchly: postponing Mini Break
[2021-07-21 15:32:13.243] [info]  Stretchly: showing Mini Break notification
[2021-07-21 15:32:23.732] [info]  Stretchly: showing window 1 of 3
[2021-07-21 15:32:23.735] [info]  Stretchly: starting Mini Break
[2021-07-21 15:32:23.879] [info]  Stretchly: showing window 3 of 3
[2021-07-21 15:32:24.011] [info]  Stretchly: showing window 2 of 3
[2021-07-21 15:32:24.819] [info]  Stretchly: finishing Mini Break // here. I have to quit Sketchly and restart
[2021-07-21 15:32:54.606] [info]  Stretchly: initializing...
[2021-07-21 15:32:54.614] [info]  Stretchly: loading preferences
[2021-07-21 15:32:54.643] [info]  Stretchly: starting Idle time monitoring
[2021-07-21 15:32:54.644] [info]  Stretchly: starting Do Not Disturb monitoring

**Comment by hovancik** [comment]
@imageslr can you try to reproduce with 2 monitors only? I don't have 3 to test with

**Comment by imageslr** [comment]
@hovancik  Hi I've reproduced this bug with only mac. I first skipped to a mini break and instantly press `cmd + x` many times, to postpone the break. 

Here is the log:
```
[2021-07-25 23:17:51.809] [info]  Stretchly: skipping to Mini Break
[2021-07-25 23:17:52.294] [info]  Stretchly: showing window 1 of 1
[2021-07-25 23:17:52.298] [info]  Stretchly: starting Mini Break
[2021-07-25 23:17:52.385] [info]  Stretchly: postponing Mini Break

// restart stretchly 
[2021-07-25 23:18:13.631] [info]  Stretchly: initializing...
[2021-07-25 23:18:13.643] [info]  Stretchly: loading preferences
[2021-07-25 23:18:13.657] [info]  Stretchly: starting Idle time monitoring
[2021-07-25 23:18:13.660] [info]  Stretchly: starting Do Not Disturb monitoring
```

Here is my config.json:

```json
{
	"microbreakDuration": 20000,
	"microbreakInterval": 2400000,
	"breakDuration": 300000,
	"breakInterval": 2,
	"breakNotification": true,
	"microbreakNotification": true,
	"breakNotificationInterval": 30000,
	"microbreakNotificationInterval": 10000,
	"microbreak": true,
	"break": true,
	"microbreakStrictMode": false,
	"breakStrictMode": false,
	"morningHour": 6,
	"microbreakPostpone": true,
	"breakPostpone": true,
	"microbreakPostponeTime": 120000,
	"breakPostponeTime": 300000,
	"microbreakPostponesLimit": 1,
	"microbreakPostponableDurationPercent": 30,
	"breakPostponesLimit": 1,
	"breakPostponableDurationPercent": 30,
	"mainColor": "#478484",
	"transparentMode": true,
	"opacity": 0.9,
	"audio": "wind-chime",
	"volume": 1,
	"fullscreen": true,
	"ideas": true,
	"naturalBreaks": true,
	"naturalBreaksInactivityResetTime": 300000,
	"allScreens": true,
	"useIdeasFromSettings": false,
	"language": "zh-CN",
	"notifyNewVersion": true,
	"isFirstRun": false,
	"posLatitude": 0,
	"posLongitude": 0,
	"useMonochromeTrayIcon": true,
	"useMonochromeInvertedTrayIcon": false,
	"silentNotifications": false,
	"monitorDnd": true,
	"microbreakStartSoundPlaying": false,
	"breakStartSoundPlaying": false,
	"themeSource": "system",
	"endBreakShortcut": "CmdOrCtrl+X",
	"breakWindowWidth": 0.85,
	"breakWindowHeight": 0.85,
	"checkNewVersion": true,
	"breakIdeas": [
		{
			"data": [
				"Not alone",
				"Do you find it hard to take a break alone? Try to do it with a co-worker. Aside from making it easier to stick to breaks, you will have a chance to get to know them better. Taking breaks together increases productivity."
			],
			"enabled": true
		},
		{
			"data": [
				"Step away",
				"Do you ever notice how your brain can figure things out by itself? All it takes is to step away from the computer and take a break to think about something totally unrelated."
			],
			"enabled": true
		},
		{
			"data": [
				"Microbreaks",
				"Rest is a key component in ensuring the performance of the musculoskeletal system. Frequent breaks can decrease the duration of a task and help lower the exposure to ergonomic injury risk."
			],
			"enabled": true
		},
		{
			"data": [
				"Meditation",
				"Research studies suggest that mindfulness-based exercises help decrease anxiety, depression, stress, and pain, and help improve general health, mental health, and quality of life. Not sure how to start? There are numerous apps to help you out."
			],
			"enabled": true
		},
		{
			"data": [
				"Blink",
				"Looking at screens for a long time causes you to blink less, thus exposing your eyes to the air. Blink rapidly for a few seconds to refresh the tear film and clear dust from the eye surface."
			],
			"enabled": true
		},
		{
			"data": [
				"Ergonomics",
				"Improper height and angle of the keyboard, mouse, monitor or working surface can cause health problems. Take some time to read about desk ergonomics."
			],
			"enabled": true
		},
		{
			"data": [
				"Move",
				"There are a lot of ways you can exercise within your office. Try marching in place or doing desk push-ups."
			],
			"enabled": true
		},
		{
			"data": [
				"Change",
				"Do you have a stability ball or standing work desk? Consider replacing your desk chair with them for a while."
			],
			"enabled": true
		},
		{
			"data": [
				"Notice",
				"Are you daydreaming or having trouble focusing? It is a sign that you need to take a break."
			],
			"enabled": true
		},
		{
			"data": [
				"Tech",
				"How about taking a no-tech walk?"
			],
			"enabled": true
		},
		{
			"data": [
				"Metabolism",
				"Emerging research shows that sitting for long periods of time contributes to risk of metabolic syndrome, heart attack and stroke risk and overall death risk, among others. Taking regular walking breaks can help your circulation, working to counteract some of those problems."
			],
			"enabled": true
		},
		{
			"data": [
				"Active Meetings",
				"How about moving meetings from the conference room to the concourse? Walking not only burns calories but it may even foster a sense of collaboration."
			],
			"enabled": true
		},
		{
			"data": [
				"Fruit",
				"Take your time and eat some fruit. Slowly. Notice the flavor, the texture, the freshness."
			],
			"enabled": true
		},
		{
			"data": [
				"Bathrooms",
				"Walk to the farthest bathroom in the worksite facility when going to the restroom."
			],
			"enabled": true
		},
		{
			"data": [
				"Coffee break",
				"Going on coffee break? Consider doing a 5-minute walk every time you go for one."
			],
			"enabled": true
		},
		{
			"data": [
				"Colleagues",
				"Do not email or message office colleagues, walk to their desks to communicate with them."
			],
			"enabled": true
		},
		{
			"data": [
				"Learning",
				"In a study of healthy volunteers, NIH researchers found that taking short breaks, early and often, may help our brains learn new skills."
			],
			"enabled": true
		},
		{
			"data": [
				"Exercise",
				"Evidence suggests small amounts of regular exercise can bring dramatic health benefits, including measurably reducing stress."
			],
			"enabled": true
		},
		{
			"data": [
				"Repeat",
				"Have you found your stretch-ly-routine? Do not forget to repeat it for more than once to better fight effects of prolonged sitting."
			],
			"enabled": true
		},
		{
			"data": [
				"Wrist and forearm",
				"Extend your arms with the palms facing towards you, then slowly rotate the hands four times clockwise, then four times counter-clockwise."
			],
			"enabled": true
		},
		{
			"data": [
				"Back stretching",
				"Join your hands behind your head, then lift them together up above your head ending with your palms facing upward."
			],
			"enabled": true
		},
		{
			"data": [
				"Mobilize",
				"For every thirty minutes of stagnation, you should have at least one minute of stimulation."
			],
			"enabled": true
		},
		{
			"data": [
				"7 Minute Workout",
				"This workout packs in a full-body exercise routine in a fraction of the time. But as with any excercise, be careful. There are numerous apps to get you started."
			],
			"enabled": true
		},
		{
			"data": [
				"Pulse",
				" Raise your pulse rate to 120 beats per minute for 20 straight minutes four or five times a week doing anything you enjoy. Regularly raising your heart rate results in improved cardiovascular health."
			],
			"enabled": true
		},
		{
			"data": [
				"Take the stairs ",
				"Studies have shown that stair climbing, which is considered vigorous-intensity physical activity, burns more calories per minute than jogging."
			],
			"enabled": true
		},
		{
			"data": [
				"Make art",
				"Art therapy is known to have great mental health benefits, especially when it comes to stress management. How about writing a quick poem, taking a picture or painting something small?"
			],
			"enabled": true
		},
		{
			"data": [
				"Declutter",
				"A clean space helps your focus at work and is often linked to positive emotions like happiness."
			],
			"enabled": true
		},
		{
			"data": [
				"Lunch outside",
				"Nature is linked to positive emotions and decreased stress and anxiety. Whenever possible, try to take your daily lunch break outside, surrounded by some greenery."
			],
			"enabled": true
		},
		{
			"data": [
				"Public transport",
				"If you use public transport regularly, you can stand instead of sitting. If it is possible, try to replace as many of your daily trips as possible with walking or cycling."
			],
			"enabled": true
		},
		{
			"data": [
				"Yawning",
				"Yawning can be really helpful, as it produces tears to help moisten and lubricate the eyes."
			],
			"enabled": true
		},
		{
			"data": [
				"Focus change",
				"Hold one finger close to the eye an focus on it. Slowly move the finger away, focus far into the distance and then back to the finger. Bring the finger back and focus on something far away."
			],
			"enabled": true
		},
		{
			"data": [
				"Palming",
				"While seated, brace elbows on the desk and close to the desk edge. Let your weight fall forward and cup hands over eyes. Close your eyes and inhale slowly through nose and hold for few seconds. Continue deep breathing."
			],
			"enabled": true
		},
		{
			"data": [
				"Hand squeezes",
				"Squeeze a pair of balled-up socks or a soft rubber ball, hold for 5 seconds. Repeat whole process few times."
			],
			"enabled": true
		}
	],
	"microbreakIdeas": [
		{
			"data": "Go grab a glass of water.",
			"enabled": true
		},
		{
			"data": "Slowly look all the way left, then right.",
			"enabled": true
		},
		{
			"data": "Slowly look all the way up, then down.",
			"enabled": true
		},
		{
			"data": "Close your eyes and take few deep breaths.",
			"enabled": true
		},
		{
			"data": "Close your eyes and relax.",
			"enabled": true
		},
		{
			"data": "Stretch your legs.",
			"enabled": true
		},
		{
			"data": "Stretch your arms.",
			"enabled": true
		},
		{
			"data": "Is your sitting posture correct?",
			"enabled": true
		},
		{
			"data": "Slowly turn head to side and hold for 10 seconds.",
			"enabled": true
		},
		{
			"data": "Slowly tilt head to side and hold for 5-10 seconds.",
			"enabled": true
		},
		{
			"data": "Stand from your chair and stretch.",
			"enabled": true
		},
		{
			"data": "Refocus your eyes on an object at least 20 meters away.",
			"enabled": true
		},
		{
			"data": "Take a moment to think about something you appreciate.",
			"enabled": true
		},
		{
			"data": "Take a moment to smile at being alive.",
			"enabled": true
		},
		{
			"data": "A truly ergonomic workstation is one that you regularly push away from.",
			"enabled": true
		},
		{
			"data": "Close your eyes and count your breaths.",
			"enabled": true
		},
		{
			"data": "Close your eyes and name the things you hear.",
			"enabled": true
		},
		{
			"data": "Place your fingertips on your shoulders. Roll your shoulders forward for 10 seconds, then backward.",
			"enabled": true
		},
		{
			"data": "Raise your right arm, stretch it over your head to the left, and hold for 10 seconds. Repeat on the other side.",
			"enabled": true
		},
		{
			"data": "With your right hand, grab each finger of your left hand in turn and squeeze. Repeat on the other side.",
			"enabled": true
		},
		{
			"data": "Stand up and do a lunge. Hold for 10 seconds, then do the other leg.",
			"enabled": true
		},
		{
			"data": "Close your eyes and simply notice whatever arises in current moment, without judgement.",
			"enabled": true
		},
		{
			"data": "One should focus every 20 minutes for 20 seconds on an object that is 20 feet distance.",
			"enabled": true
		},
		{
			"data": "If you need help, ask for it.",
			"enabled": true
		},
		{
			"data": "Do one thing at a time.",
			"enabled": true
		},
		{
			"data": "Is your attention spent wisely?",
			"enabled": true
		},
		{
			"data": "Change your sitting posture.",
			"enabled": true
		},
		{
			"data": "Expose your eyes to natural light.",
			"enabled": true
		},
		{
			"data": "With your eyes closed, slowly and gently move eyes up to the ceiling, then slowly down to the floor.",
			"enabled": true
		},
		{
			"data": "With your eyes closed, slowly and gently move eyes to the left, then slowly to the right.",
			"enabled": true
		},
		{
			"data": "Shake your hands out to get some relief.",
			"enabled": true
		},
		{
			"data": "One at a time, touch the tip of each finger to the tip of your thumb so they make an O-shape.",
			"enabled": true
		},
		{
			"data": "Make a fist and then slide your fingers up until they point toward the ceiling, like you're telling someone to stop.",
			"enabled": true
		},
		{
			"data": "Make a fist and then fan your fingers out and stretch them as far as you can.",
			"enabled": true
		}
	],
	"showBreaksAsRegularWindows": false,
	"appExclusions": [],
	"pauseForSuspendOrLock": true,
	"pauseBreaksShortcut": "",
	"resumeBreaksShortcut": "",
	"screen": "primary"
}
```

**Comment by imageslr** [comment]
![image](https://user-images.githubusercontent.com/17822028/128120852-bc9cc820-da45-4bfb-bd0c-fe6fc8aabbf8.png)

**Comment by hovancik** [comment]
Thanks! I'll try to reproduce next week, I am on holidays and don't have a mac with me.

**Comment by imageslr** [comment]
Thank you. This is not an emergency bug. I just paste more info to help. Enjoy your holiday!

**Comment by hovancik** [comment]
@imageslr what's the number of monitors you have? 

I have my macbook plus one external monitor but can't reproduce this ;/ I am trying by skipping to the next Mini/Long Break and pushing CmdX right away

**Comment by imageslr** [comment]
@hovancik I have no external monitor now. And I indeed reproduce this by skipping to the next mini break and pushing cmdx right away just now. Have you set "Full screen"?

**Comment by hovancik** [comment]
Ah, thanks,I was not in fullscreen.

**Comment by hovancik** [comment]
So, I can reproduce if I do Cm+X between window being created but not really fully loaded. All seems fine when I postpone after all is loaded. There is probably some short interval between creating the window and window being ready when closing it breaks something. 

How it is for you? Do you ever get error when postponing after break is fully loaded?

I guess I could change it in a way that shortcut is available only after windows is fully loaded.  I would need to check how it works eexactly.

**Comment by imageslr** [comment]
> How it is for you? Do you ever get error when postponing after break is fully loaded?

No. This error only happens when I press the hot key very quickly. 

It has no influence on me. This is not an emergency problem. Take your time! Thanks :)

**Comment by r3a1d3a1** [comment]
This often happens to me too...
In other words, I see the notification for a break, I lock my Mac (single monitor) a few seconds later, and when I come back later on, I've a really hard time trying to click on anything to be able to kill Stretchly.

**Comment by hovancik** [comment]
New version will have updated behavior when starting new break so let's see whether that helps

**Comment by r3a1d3a1** [comment]
(By the way, outside the scope of this thread, I found it misleading for 61 seconds to be considered "about 2 minutes", so I fixed and refactored the code [here](https://github.com/hovancik/stretchly/commit/3a0f64b036179778b50d6f03145775ec97e77265). 50 lines down to 25 :) Feel free to incorporate it if you like.)

**Comment by hovancik** [comment]
@r3a1d3a1 what part of the app do you mean? Break window or tray?

**Comment by r3a1d3a1** [comment]
Both

**Comment by hovancik** [comment]
Please let me know if the original issue still happens in the latest window. Discussion went offtopic a bit, so closing this one.
